### PR TITLE
Feat/fix java doc

### PR DESCRIPTION
**Pull request by xixingya**
this pull request fix the wrong java doc with issue https://github.com/spring-projects/spring-framework/issues/27269

### CODE REVIEW

**Comment by xixingya** [comment]
@rstoyanchev can you help to review the pull request?

File: spring-webmvc/src/main/java/org/springframework/web/servlet/config/annotation/PathMatchConfigurer.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ public PathMatchConfigurer addPathPrefix(String prefix, Predicate<Class<?>> pred
 	 * @deprecated as of 5.2.4. See class-level note in
 	 * {@link RequestMappingHandlerMapping} on the deprecation of path extension
 	 * config options. As there is no replacement for this method, in 5.2.x it is
-	 * necessary to set it to {@code false}. In 5.3 the default changes to
+	 * necessary to set it to {@code false}. by default it is set to
```
Comment (snicoll) [inline spring-webmvc/src/main/java/org/springframework/web/servlet/config/annotation/PathMatchConfigurer.java:129]: The curent javadoc reads ok to me. What lead you to such a change?

Comment (xixingya) [reply spring-webmvc/src/main/java/org/springframework/web/servlet/config/annotation/PathMatchConfigurer.java:129]: https://github.com/spring-projects/spring-framework/issues/27269 see this issues

Comment (xixingya) [reply spring-webmvc/src/main/java/org/springframework/web/servlet/config/annotation/PathMatchConfigurer.java:129]: I have misunderstand sorry

**Pull request closed without merging**